src/metabase/api/database.clj

```diff
@@ -1073,9 +1075,10 @@
                            (filter (partial can-read-schema? id) schemas)))]
     (->> (t2/select-fn-set :schema Table
                            :db_id id :active true
-                         ;; a non-nil value means Table is hidden -- see [[metabase.models.table/visibility-types]]
-                           :visibility_type nil
-                           {:order-by [[:%lower.schema :asc]]})
+                           {:order-by [[:%lower.schema :asc]]
```

Comment:
how about using a merge for this? sth like

```clojure
(merge {:order-by ...} (when-not include_hidden {:where ...}))
```

Reply:
Yes, I like it better. Thanks